src/test/kotlin/org/rust/ide/actions/mover/RsStatementUpDownMoverTest.kt

```diff
@@ -0,0 +1,421 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.ide.actions.mover
+
+class RsStatementUpDownMoverTest : RsStatementUpDownMoverTestBase() {
+    fun `test straightline down`() = moveDown("""
+        fn main() {
+            1;
+            2/*caret*/;
+            3;
+        }
+    """, """
+        fn main() {
+            1;
+            3;
+            2/*caret*/;
+        }
+    """)
+
+    fun `test straightline up`() = moveUp("""
+        fn main() {
+            1;
+            2;
+            3/*caret*/;
+        }
+    """, """
+        fn main() {
+            1;
+            3/*caret*/;
+            2;
+        }
+    """)
+
+    fun `test straightline down out of body`() {
+        val code = """
+            fn main() {
+                1;
+                2;
+                3/*caret*/;
+            }
+        """
+        UpDownMoverTestMarks.moveOutOfBody.checkHit {
+            moveDownForbidden(code)
+        }
+    }
+
+    fun `test straightline up out of body`() {
+        val code = """
+            fn main() {
+                1/*caret*/;
+                2;
+                3;
+            }
+        """
+        UpDownMoverTestMarks.moveOutOfBody.checkHit {
```

Comment:
It seems almost all tests (looks like except movers tests) uses the following style (or something similar)
```kotlin
private fun doTest(someArgs, testmark: Testmark? = null) {
    val action = { someActions }
    testmark?.checkHit(action) ?: action()
}
```
Let's unify using `Testmark` in these tests too. It would be cool if you refactor other mover tests as well

Reply:
Done.

Reply:
Thanks!